linker/shared/src/main/scala/org/scalajs/linker/backend/webassembly/Instructions.scala

```diff
@@ -0,0 +1,395 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.linker.backend.webassembly
+
+import org.scalajs.ir.Position
+
+import Identitities._
+import Types._
+
+/** WebAssembly instructions.
+ *
+ *  @see
+ *    [[https://webassembly.github.io/spec/core/syntax/instructions.html]]
+ */
+object Instructions {
+
+  /** A WebAssembly `expr`. */
+  sealed case class Expr(instr: List[Instr])
```

Comment:
```suggestion
  final case class Expr(instr: List[Instr])
```

Reply:
Done, as well as on all other `case class`es of this file.